CONTRIBUTING.md

```diff
@@ -24,8 +24,33 @@ For more on how 10up writes and manages code, check out our [10up Engineering Be
 
 ## Workflow
 
-The `develop` branch is the development branch which means it contains the next version to be released. `trunk` contains the latest released version.  Always work on the `develop` branch and open up PRs against `develop`.
+The `develop` branch is the development branch which means it contains the next version to be released. `trunk` contains the latest released version.
+
+### Feature branches
+
+1. From the `develop` branch create a new feature branch with the following naming structure: `feature|fix/[github-issue-number]--[branch-description]` (i.e. `feature/10--accordion-component-build`).
+2. Do work on the `feature` or `fix` branch. When finished, submit a pull request to merge your work back into the `develop` branch. In the PR, link to the issue (see the sidebar options on the right).
+3. If there are conflicts between your branch and the code that now exists on `develop` branch, merge `develop` into your branch to reintegrate new changes and resolve any conflicts.
+4. Assign a minimum on 1 reviewer to the pull request. At least one engineer must review all PRs.
+5. If any updated components (`/packages`) that will need to be relased to npm exist, be sure to update package version. numbers in their respective `package.json` files.
+6. Once PR is approved and all tests pass, the creator of the PR performs the merge into `develop`.
+7. Pull down the develop branch locally and test updates to verify everything is working correctly.
+
+
+### Release instructions
+_NOTE: Release is this instance is referring to the repository version history - not the npm package(s)_
+
+1. Create a new issue in github to track the release. Name the release version in the github issue. Refer to [releases](https://github.com/10up/component-library/releases) for the latest release.
+2. Create a branch from `develop` named `release/[github-issue-number]-[release-number]`.
+3. With the release branch checked out locally, in the `/docs` directory run `bundle install` and `bundle exec jekyll serve`. Verify everything is working correctly.
+4. Create a pull request in github to merge the `release/[github-issue-number]-[release-number]` branch into `trunk`. Assign one reviewer to the PR. Add "Component Libarary Release [release number]" and "CHANGELOG" updates to the description.
+5. Once branch is approved, all tests pass, and merged on github, switch to the `trunk` branch locally and pull the changes you just merged.
+6. Verify changes are working correctly locally and, once the deploy is complete, on [Baseline](https://baseline.10up.com/components).
+7. If any updated components (`/packages`) that will need to be relased to npm exist, be sure to update package version numbers in their respective `package.json` files (this should have been completed in the branch workflow - see above).
+8. If any package.json files need updating, be sure to commit the updates and push to the remote `trunk` branch.
```

Comment:
I think this should be done as part of the `release` branch instead of pushing to `trunk` directly.

Reply:
Documentation updated, please review @nicholasio